app/services/classification/base.rb

```diff
@@ -146,7 +146,7 @@ def path_elements(path = nil)
 
     def validate_lookup_value(key, value)
       lookup_value = key.lookup_values.build(:value => value)
-      return true if lookup_value.send(:validate_list) && lookup_value.send(:validate_regexp)
+      return true if lookup_value.send(:validate_value)
```

Comment:
things is, it should. this code shouldn't be here, but I don't know how to move it away...